### PR TITLE
Add postman collection for access rule management.

### PR DESCRIPTION
**Pull request by tunacicek**
## Description
Add postman collection for access rule management.
Adjust default wildcard types

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/development/postman/tractusx-dtr-access-rule-management-api-v1.0-collection.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
+{
+	"info": {
+		"_postman_id": "47f1eefc-bd84-4c6b-9c30-2ff696b1dde0",
+		"name": "tractusx-dtr-access-rule-management-api-v1.0 Copy",
```
Comment (nagyesta) [inline docs/development/postman/tractusx-dtr-access-rule-management-api-v1.0-collection.json:4]: Is it intentional that the name contains Copy?

Comment (istvan-nagy-epam) [inline docs/development/postman/tractusx-dtr-access-rule-management-api-v1.0-collection.json:4]: Is it intentional that the name contains Copy?

Comment (tunacicek) [reply docs/development/postman/tractusx-dtr-access-rule-management-api-v1.0-collection.json]: Done

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**